Source/Core/Core/NetPlayServer.cpp

```diff
@@ -802,6 +846,25 @@ void NetPlayServer::SetNetSettings(const NetSettings& settings)
 }
 
 // called from ---GUI--- thread
+bool NetPlayServer::RequestStartGame()
+{
+  if (m_sync_save_data)
```

Comment:
These can be joined as:

```cpp
if (m_sync_save_data && !SyncSaveData())
```

Reply:
No it can't, as we don't want the else block (which starts the game) to be executed is syncing is enabled but it failed.

Reply:
ah, true, good catch